pandas/tests/plotting/test_frame.py

```diff
@@ -367,6 +367,65 @@ def test_subplots(self):
             for ax in axes:
                 assert ax.get_legend() is None
 
+    def test_groupby_boxplot_sharey(self):
+        # https://github.com/pandas-dev/pandas/issues/20968
+        # sharey can now be switched check whether the right
+        # pair of axes is turned on or off
+
+        def _assert_ytickslabels_visibility(axes, expected):
+            for ax, exp in zip(axes, expected):
+                self._check_visible(ax.get_yticklabels(), visible=exp)
+
+        df = DataFrame({'a': [-1.43, -0.15, -3.70, -1.43, -0.14],
+                        'b': [0.56, 0.84, 0.29, 0.56, 0.85],
+                        'c': [0, 1, 2, 3, 1]},
+                       index=[0, 1, 2, 3, 4])
+
+        # standart behavior
```

Comment:
standard

Reply:
what is 'standard' mean here, can you update the comment

Reply:
Done.